Wikipedia/Code/NSDateFormatter+WMFExtensions.h

```diff
@@ -39,6 +39,6 @@
 
 + (instancetype)wmf_mediumDateFormatterWithoutTime;
 
-+ (instancetype)wmf_hyphenatedYearMonthDayFormatter;
++ (instancetype)wmf_englishHyphenatedYearMonthDayFormatter;
```

Comment:
yup, you can see the implementation, but the tests also verify this behavior (now that we have EN and AR language schemes)